msg/CMakeLists.txt

```diff
@@ -107,6 +107,7 @@ set(msg_files
 	sensor_preflight.msg
 	sensor_selection.msg
 	servorail_status.msg
+	sih.msg
```

Comment:
I'd prefer a more descriptive name, like `hardware_simulator_data`?

Reply:
The sih messages are just for debug purpose (i.e. for me).
The information contains in sih.msg is a combination of the 3 messages:
- actuator_outputs
- vehicle_global_position_groundtruth
- vehicle_attitude_groundtruth

So if needed I can remove sih.msg.

Reply:
Let's remove it before merge in that case.

Reply:
done